File: LeetCode/Easy/125-valid-palindrome/try-1.java
/*
 * Problem #125: Valid Palindrome
 * Difficulty: Easy
 * Submission: Try 1
 * status: Accepted
 * Language: java
 * Date: 7/14/2026, 12:47:23 PM
 * Link: https://leetcode.com/problems/valid-palindrome/
 */

class Solution {
    public boolean isPalindrome(String s) {
        int left = 0;
        int right = s.length() - 1;

        while (left < right) {

            while (left < right && !Character.isLetterOrDigit(s.charAt(left)))
                left++;

            while (left < right && !Character.isLetterOrDigit(s.charAt(right)))
                right--;

            if (Character.toLowerCase(s.charAt(left)) !=
                Character.toLowerCase(s.charAt(right)))
                return false;

            left++;
            right--;
        }

        return true;
    }
} 
   

File: Codeforces/318/A/try-1.py
/*
 * Problem : Even Odds
 * Difficulty: Easy
 * Submission: Try 1
 * status: Accepted
 * Language: Python 3
 * Date: 9/5/2026, 7:46:53 PM
 * Link: https://codeforces.com/contest/318/problem/A
 */

n, k = map(int, input().split())

odd_count = (n + 1) // 2

if k <= odd_count:
    answer = 2 * k - 1
else:
    answer = 2 * (k - odd_count)

print(answer)
